tests/fixtures/js-stubs/builtInSymbolSources.js

```diff
@@ -0,0 +1,65 @@
+export function BuiltInSymbolSources(params = []) {
```

Comment:
Should this be a `.ts` file?

Reply:
no... Our test Stubs usually have the `.js` extension